lib/runners/processor/cppcheck.rb

```diff
@@ -111,6 +114,10 @@ def language
       config_linter[:language].then { |lang| lang ? ["--language=#{lang}"] : [] }
     end
 
+    def suppressions_list
+      (config_linter[:'suppressions-list'] || DEFAULT_SUPPRESSIONS_PATH).then { |file| (current_dir / file).exist? ? ["--suppressions-list=#{file}"] : [] }
```

Comment:
```suggestion
      (config_linter[:'suppressions-list'] || DEFAULT_SUPPRESSIONS_PATH).then { |file| ["--suppressions-list=#{file}"] }
```

The existence check is not needed for this case.

Reply:
OK, I removed it.